app/Main.hs

```diff
@@ -104,12 +120,43 @@ copyStaticFiles = do
     void $ forP filepaths $ \filepath ->
         copyFileChanged ("site" </> filepath) (outputFolder </> filepath)
 
+formatDate :: String -> String
+formatDate humanDate = toIsoDate parsedTime
+  where
+    parsedTime =
+      parseTimeOrError True defaultTimeLocale "%b %e, %Y" humanDate :: UTCTime
+
+rfc3339 :: Maybe String
+rfc3339 = Just "%H:%M:SZ"
+
+toIsoDate :: UTCTime -> String
+toIsoDate = formatTime defaultTimeLocale (iso8601DateFormat rfc3339)
+
+buildFeed :: [Post] -> Action ()
+buildFeed posts = do
+  now <- liftIO getCurrentTime
+  let atomData =
+        AtomData
+          { title = "<site-title-here>"
```

Comment:
Please use values from the `siteMeta` constant here; not much point in the user entering all their data twice 😄

Reply:
good call!